server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodReplicationTest.java

```diff
@@ -138,20 +139,20 @@ public void testReplicatedPutWithTopologyChanges(Method m) {
       }
 
       resp = clients().get(0)
-                      .put(k(m), 0, 0, v(m, "v5-"), (byte) 2, ClusterCacheStatus.INITIAL_TOPOLOGY_ID + 2 * nodeCount() + 2);
+            .put(k(m), 0, 0, v(m, "v5-"), (byte) 2, 0);
```

Comment:
```suggestion
            .put(k(m), 0, 0, v(m, "v5-"), (byte) 2, serverClientTopologyId);
```

Reply:
The test wants the incorrect topology as the line 145 verifies a topology was sent back. We don't want to pass the correct topology, otherwise it will be null.

Reply:
I'm not understanding the test then. You're stopping a server in line 137; even if you use `serverClientTopologyId`, it is outdated and should get the new topology id.